### ISSUE TITLE
bug: Property 'inert' does not exist on type 'HTMLAttributes<HTMLXElement>'

### ISSUE THREAD

**Issue by jcfranco** [ionic-team/stencil#5071]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

4.7.0

### Current Behavior

Using an element with `inert` displays the following error when running the Stencil build:

```
Type '{ inert: true; }' is not assignable to type 'HTMLAttributes<HTMLDivElement>'.Property 'inert' does not
       exist on type 'HTMLAttributes<HTMLDivElement>'.
```

### Expected Behavior

To be able to use `inert` without any TS warnings/errors.

### Steps to Reproduce

See repro case's README.

### Code Reproduction URL

https://github.com/jcfranco/stencil-inert-missing-from-types

**Comment by jcfranco** [comment]
Is it possible to work around this with declaration merging or does Stencil always have to update its types?

**Comment by FabioGimmillaro** [comment]
Hi @jcfranco 

As a workaround do this:
```tsx
<div
    {
      ...{ inert: true }
    }
>
   ...
</div>
```

**Comment by alicewriteswrongs** [comment]
The fix for this issue has been released as a part of today's [Stencil v4.8.0 release](https://github.com/ionic-team/stencil/releases/tag/v4.8.0)!
